Tao.md

```diff
@@ -29,10 +29,10 @@ document but an informal and informational overview.
 
 ## Table of Contents
 
-1. <a href="#introduction">Introduction</a><br>
+1 <a href="#introduction">Introduction</a><br>
```

Comment:
More readable:

```suggestion
1 [Introduction](#introduction)<br>
```

Likewise for other links.

Reply:
I would rather fix content and leave the markdown fixups for a separate PR.  See #42

Reply:
ACK